packages/airnode-node/src/reporting/heartbeat.ts

```diff
@@ -56,16 +42,18 @@ export async function reportHeartbeat(state: CoordinatorState): Promise<PendingL
   const httpGatewayUrl = getHttpGatewayUrl(config);
   const httpSignedDataGatewayUrl = getHttpSignedDataGatewayUrl(config);
 
-  const heartbeatPayload = {
+  const timestamp = Date.now();
+
+  const heartbeatPayload = JSON.stringify({
```

Comment:
Hmm. Maybe it would be wise to make sure that the payload content fields are always sorted. `JSON.stringify` doesn't do that. Maybe something like [`json-stable-stringify`](https://github.com/ljharb/json-stable-stringify] would be better. It's not like `JSON.stringify` will stringify it randomly, the fields will be in the order they are presented in when creating the object. But just to be on the safe side?

Reply:
If we serialise it like this we will always get a consistent result.
I have encountered a situation once before where the keys were serialised out of alphabetical order (I think they're serialised in the order they're set), but serialising a serialised payload gets rid of that issue and any other potential issues as you're comparing the signature with a byte array effectively. Adding an additional library to ensure serialisation ordering consistency seems unnecessary to me and introduces a new dependency.